tests/test_wp.py

```diff
@@ -1,61 +1,46 @@
+from pytest import approx
+
 from happytransformer import HappyWordPrediction
 from happytransformer.happy_word_prediction import WordPredictionResult
 
 
 def test_mwp_basic():
-    happy_mwp = HappyWordPrediction()
-    result = happy_mwp.predict_mask(
-        "Please pass the salt and [MASK]",
-    )
-    answer = [WordPredictionResult(token_str="pepper", score=0.2664579749107361)]
-    assert result == answer
+    MODELS = [
+        ('DISTILBERT','distilbert-base-uncased','pepper'),
+        ('BERT','bert-base-uncased','.'),
+        ('ALBERT','albert-base-v2','garlic')
+    ]
+    for model_type,model_name,top_result in MODELS:
+        happy_mwp = HappyWordPrediction(model_type,model_name)
+        results = happy_mwp.predict_mask(
+            "Please pass the salt and [MASK]",
+        )
+        result = results[0]
+        assert result.token == top_result
 
 
 def test_mwp_top_k():
-    happy_mwp = HappyWordPrediction()
+    happy_mwp = HappyWordPrediction('DISTILBERT','distilbert-base-uncased')
     result = happy_mwp.predict_mask(
         "Please pass the salt and [MASK]",
         top_k=2
     )
-    answer = [WordPredictionResult(token_str='pepper', score=0.2664579749107361),
-              WordPredictionResult(token_str='vinegar', score=0.08760260790586472)]
+    answer = [
+        WordPredictionResult(token='pepper', score=approx(0.2664579749107361,0.01)),
```

Comment:
My IDE doesn't like this. It says "Expected type float but got type ApproxBase." It still works though, so we don't have to change it right away.

Reply:
We shouldn't fix this. As far as users are concerned the score is always a float.

Reply:
okay